dali/kernels/reduce/reduce_setup_utils.h

```diff
@@ -148,15 +148,33 @@ inline void CheckAxes(span<const int> axes, int ndim) {
   assert(ndim >= 0 && ndim <= 64);
   uint64_t mask = 0;
   for (auto a : axes) {
-    if (a < 0 || a >= ndim)
-      throw std::out_of_range(make_string("Axis index out of range: ", a, " not in 0..", ndim-1));
+    if (a < -ndim || a >= ndim)
+      throw std::out_of_range(make_string("Axis index out of range: ", a, " not in ", -ndim, "..", ndim-1));
```

Comment:
How about applying mathematical range notation?
```suggestion
      throw std::out_of_range(make_string("Axis index out of range: ", axis, " not in [", -ndim(), ", ", ndim() - 1, "]"));
```

Reply:
Done